apps/ex_nvr_web/assets/js/webrtc.js

```diff
@@ -0,0 +1,54 @@
+import { Socket } from "phoenix"
+import { WebRTCEndpoint } from "@jellyfish-dev/membrane-webrtc-js"
+
+function getWebSocketEndpoint() {
+    const protocol = window.location.protocol === 'https:' ? 'wss://' : 'ws://'
+    const hostname = window.location.hostname
+    const port = window.location.port ? `:${window.location.port}` : ''
+    
+    return `${protocol}${hostname}${port}`
+}
+
+window.onload = function (_event) {
+    const player = document.getElementById("webRtcPlayer")
+    const deviceId = player.dataset.device
+    const webrtc = new WebRTCEndpoint()
+    
+    const socket = new Socket(getWebSocketEndpoint() + "/socket", {params: {}})
```

Comment:
Is the `getWebSocketEndpoint` necessary ? can't we just use the url `/socket` and let it infer the type of connection ?